components/CatsList/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
we don't need to import `React` here

```suggestion
import { useState, useEffect } from 'react';
```